Domain/sdLitica.Analytics/AnalyticsOperation.cs

```diff
@@ -0,0 +1,48 @@
+﻿using sdLitica.Entities.Abstractions;
+using System;
+
+namespace sdLitica.Analytics
+{
+    //  At this moment, also used as its model in requests, messages. 
+
+    /// <summary>
+    /// Entity for metadata of analytics operation.
+    /// </summary>
+    public class AnalyticsOperation: Entity
+    {
+
+        public AnalyticsOperation()
+        {
+        }
+
+        public void SetId()
+        {
+            Id = Guid.NewGuid();
+        }
+
+
+        public Guid Id { get; set; } // should have 'protected set'
+
+        /// <summary>
+        /// Name of operation to be performed. 
+        /// </summary>
+        public string OpName { get; set; } // should have 'protected set'
+
+        /// <summary>
+        /// Id of time-series on which operation is performed
+        /// </summary>
+        public string TsId { get; set; }
+
+
+        /// <summary>
+        /// Current status of operation. 1 - complete, 0 - in process, -1 - error
+        /// </summary>
+        public int Status { get; set; }
```

Comment:
Maybe we could use an Enum to bring some meaning in the Status property datatype.